### ISSUE TITLE
[Bug]: ts_project prints a misleading warning when running with --strategy=TsProject=sandboxed

### ISSUE THREAD

**Issue by thesayyn** [aspect-build/rules_ts#293]
### What happened?

ts_project prints a misleading error when running in one-shot worker mode. The error message printed out should only be printed when the process is running at `output-base/execroot` not `output-base/execroot/sandbox/<id>`. 

### Version

Development (host) and target OS/architectures:

Output of `bazel --version`:

Version of the Aspect rules, or other relevant rules from your
`WORKSPACE` or `MODULE.bazel` file:

Language(s) and/or frameworks involved:

### How to reproduce

```shell
run a ts_project with `--strategy=TsProject=sandboxed`
```

### Fund our work

- [ ] Sponsor our open source work by donating a [bug bounty](https://opencollective.com/aspect-build/)
